src/context/AuthContext.tsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import firebase from 'firebase/compat/app';
```

Comment:
can be more accurately 
import {createContext} from 'react'
import {User} form 'firebase...

Reply:
import {createContext} from 'react' - fixed
import {User} form 'firebase... - I do not change because it appears problem (firebase do not export this module)

Reply:
ok, good to know